### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by t-j-h** [comment]
Nice work! 
We just need a CLA in place so we can review and process the PR.

**Comment by rex4539** [comment]
According to the contributing guidelines, if the pull request is trivial (typos etc) then only the `CLA: trivial` is required, which is already in place.

This pull request is just typos so it should be fine.

**Comment by t-j-h** [comment]
It is well beyond the scope of trivial given the size of the fix.

**Comment by rex4539** [comment]
CLA is signed and sent to the Foundation.

**Comment by rex4539** [comment]
Looks like the CI failure is unrelated to this commit?

**Comment by t8m** [comment]
> Looks like the CI failure is unrelated to this commit?

Yes, unrelated.

**Comment by paulidale** [comment]
Kicked the CLA bot to remove the tag.

File: test/sslapitest.c [hunk 1]
```diff
@@ -844,7 +844,7 @@ static int test_ccs_change_cipher(void)
     size_t readbytes;
 
     /*
-     * Create a conection so we can resume and potentially (but not) use
+     * Create a conenction so we can resume and potentially (but not) use
```
Comment (t8m) [inline test/sslapitest.c:847]: Still a typo: connection.

**Comment by t8m** [comment]
If this was split into two commits - one for the actual code and documentation changes and one for changes in comments, the first commit would be applicable to 3.0 branch as well.

File: test/sslapitest.c (hunk 1 above)
Comment (rex4539) [reply test/sslapitest.c]: Fixed, thanks!

**Comment by rex4539** [comment]
I'm not very fluent in git commands. I have no idea how to do that without totally messing up the commit :)

But, please, feel free to edit my commit and do whatever you need.

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
You'd have to do mixed reset of the tree to HEAD~ and then individually git add files that would be in the first commit, commit it and then commit the rest of the changes. But anyway, I do not think it is absolutely necessary to have these typos fixed in 3.0 branch.

**Review by t-j-h — APPROVED** [review]

**Comment by t-j-h** [comment]
Happy for this into master and I don't see a need to push into the 3.0 branch - although I wouldn't object if someone does that work in a follow up PR.

**Comment by t8m** [comment]
Merged to master. Thank you for your contribution.

**Pull request closed without merging**